mbofui/app/js/controllers.js

```diff
@@ -154,16 +183,23 @@ ocellus.controller('mapController', ['$scope', '$rootScope','$filter', '$timeout
       }
     });
   };
+
+  // watch on changes to marker (event) collection and text input so that only events with
+  // the searched for text appear
   $scope.$watch("markers", function() {
     $scope.$watch('markerFilter', function(text) {
       $scope.markersFiltered = $filter('filter')($scope.markers, {
-        event: text
+        message: text
```

Comment:
I think now that the refactoring has been completed we shouldn't see any mistakes between event and message.